modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2164,7 +2167,7 @@ void findBroadAxis(const MatShape& broadShape, const MatShape& outShape, size_t&
 
     // find the first non-one element of the broadcasting shape
     axis = 0;
-    for (; axis < broadShape.size() && broadShape[axis] == 1; ++axis) {}
+    for (; axis < broadShape.size() && broadShape[axis] == 1 && broadShape[axis] == outShape[axis]; ++axis) {}
```

Comment:
I think we should fix the comment as well. And since this is a bug, the separate PR should be created to 3.4 branch instead.

Reply:
Hi @rogday, thanks for code reviewing. How about createing a separate PR just to fixed bug at `broadShape` without including the new feature at `ReduceSum`?

Reply:
Yes, that's what I meant.